docs/operations.md

```diff
@@ -123,6 +167,128 @@ stays in the database:
 
 **Default**: `30` (days)
 
+## Database option
+
+### Always keep scavenged (_Deprecation Notice_)
+
+This option ensures that the newer chunk from a scavenge operation is always kept.
+
+| Format               | Syntax                    |
+|:---------------------|:--------------------------|
+| Command line         | `--always-keep-scavenged` |
+| YAML                 | `AlwaysKeepScavenged`     |
+| Environment variable | N/A                       |
+
+### Scavenge backend page size
+
+Specify the page size of the scavenge database.The default value is 16KiB.
+
+| Format               | Syntax                         |
+|:---------------------|:-------------------------------|
+| Command line         | `--scavenge-backend-page-size` |
+| YAML                 | `ScavengeBackendPageSize`      |
+| Environment variable | N/A                            | 
```

Comment:
are the environment variables really not applicable? if so we probably made a mistake